lib/rules/require-space-after-object-keys.js

```diff
@@ -46,12 +46,13 @@ module.exports.prototype = {
         var tokens = file.getTokens();
         file.iterateNodesByType('ObjectExpression', function(node) {
             node.properties.forEach(function(property) {
-                var key = property.key;
-                var keyPos = file.getTokenPosByRangeStart(key.range[0]);
-                var colon = tokens[keyPos + 1];
-                if (colon.range[0] === key.range[1]) {
-                    errors.add('Missing space after key', key.loc.end);
-                }
+                var token = file.getFirstNodeToken(property.key);
+                errors.assert.whitespaceBetween({
+                    token: token,
+                    nextToken: file.getNextToken(token),
+                    message: 'Missing space after key',
+                    spaces: 1
```

Comment:
I should probably change the other commit as well then https://github.com/jscs-dev/node-jscs/commit/f02aa48c5c6f5b12d6bc71356c7cc354dfe23f9b#diff-e64981daea06ee3997fe34d106961f0eR79 to remove spaces